terraform/outputs.tf

```diff
@@ -81,16 +105,15 @@ output "unamaged_disk_name" {
 
 output "vm_names" {
   value = [
-           "${azurerm_virtual_machine.vm_windows_internal.name}",
-           "${azurerm_virtual_machine.vm_linux_internal.name}",
-          ]
+    "${azurerm_virtual_machine.vm_windows_internal.name}",
+    "${azurerm_virtual_machine.vm_linux_internal.name}",
+  ]
 }
 
 output "windows_vm_name" {
   value = "${azurerm_virtual_machine.vm_windows_internal.name}"
 }
 
-
 output "windows_vm_location" {
```

Comment:
Thanks for fixing the formatting inconsistencies!

Reply:
I think that was my editor running terraform fmt automatically.. but your welcome.